gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/BuildTaskHelper.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.gradle.plugin;
+
+import org.eclipse.jkube.kit.build.service.docker.DockerAccessFactory;
+import org.eclipse.jkube.kit.build.service.docker.ImagePullManager;
+import org.eclipse.jkube.kit.build.service.docker.ServiceHubFactory;
+import org.eclipse.jkube.kit.build.service.docker.access.DockerAccess;
+import org.eclipse.jkube.kit.build.service.docker.access.log.LogOutputSpecFactory;
+import org.eclipse.jkube.kit.common.JavaProject;
+import org.eclipse.jkube.kit.common.KitLogger;
+import org.eclipse.jkube.kit.config.resource.BuildRecreateMode;
+import org.eclipse.jkube.kit.config.resource.PlatformMode;
+import org.eclipse.jkube.kit.config.service.BuildServiceConfig;
+import org.eclipse.jkube.kit.config.service.EnricherManager;
+import org.eclipse.jkube.kit.config.service.JKubeServiceHub;
+
+import static org.eclipse.jkube.kit.build.service.docker.DockerAccessFactory.DockerAccessContext.DEFAULT_MAX_CONNECTIONS;
+import static org.eclipse.jkube.kit.build.service.docker.ImagePullManager.createImagePullManager;
+
+public class BuildTaskHelper {
```

Comment:
If this is going to be reused by Push tasks (or others), why are you calling it BuildTaskHelper.

Maybe something like `TaskUtil` in the `org.eclipse.jkube.gradle.plugin.task` package can be reused for other composite methods applicable to the rest of the tasks.